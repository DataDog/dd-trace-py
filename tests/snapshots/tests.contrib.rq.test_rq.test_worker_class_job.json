[[
  {
    "name": "rq.queue.enqueue_job",
    "service": "rq",
    "resource": "tests.contrib.rq.jobs.JobClass.__call__",
    "trace_id": 0,
    "span_id": 1,
    "parent_id": 0,
    "type": "worker",
    "error": 0,
    "meta": {
      "_dd.p.dm": "-0",
      "component": "rq",
      "job.func_name": "__call__",
      "job.id": "aa66430c-d949-4911-8068-a8281de22216",
      "language": "python",
      "queue.name": "q",
      "runtime-id": "dc862c6ff169400783e513c72f18a9f3",
      "span.kind": "producer"
    },
    "metrics": {
      "_dd.agent_psr": 1.0,
      "_dd.top_level": 1,
      "_dd.tracer_kr": 1.0,
      "_sampling_priority_v1": 1,
      "process_id": 55795
    },
    "duration": 1851606,
    "start": 1691167586720339321
  },
     {
       "name": "rq.worker.perform_job",
       "service": "rq-worker",
       "resource": "__call__",
       "trace_id": 0,
       "span_id": 2,
       "parent_id": 1,
       "type": "worker",
       "error": 0,
       "meta": {
         "_dd.p.dm": "-0",
         "component": "rq",
         "job.id": "aa66430c-d949-4911-8068-a8281de22216",
         "job.origin": "q",
         "job.status": "finished",
         "language": "python",
         "runtime-id": "dc862c6ff169400783e513c72f18a9f3",
         "span.kind": "consumer",
         "traceparent": "00-000000000000000038f69a710ccab3b7-33f0b1d77d565d90-01",
         "tracestate": "dd=s:1;t.dm:-0"
       },
       "metrics": {
         "_dd.top_level": 1,
         "_dd.tracer_kr": 1.0,
         "_sampling_priority_v1": 1,
         "process_id": 55795
       },
       "duration": 1770570,
       "start": 1691167586731431663
     },
        {
          "name": "rq.job.perform",
          "service": "rq-worker",
          "resource": "__call__",
          "trace_id": 0,
          "span_id": 3,
          "parent_id": 2,
          "type": "",
          "error": 0,
          "meta": {
            "_dd.p.dm": "-0",
            "component": "rq",
            "job.id": "aa66430c-d949-4911-8068-a8281de22216"
          },
          "duration": 189140,
          "start": 1691167586731959169
        }],
[
  {
    "name": "rq.queue.enqueue_job",
    "service": "rq",
    "resource": "tests.contrib.rq.jobs.JobClass.job_on_class",
    "trace_id": 1,
    "span_id": 1,
    "parent_id": 0,
    "type": "worker",
    "error": 0,
    "meta": {
      "_dd.p.dm": "-0",
      "component": "rq",
      "job.func_name": "job_on_class",
      "job.id": "9ccae7ee-b7c5-4189-a8a6-359ae2ed8570",
      "language": "python",
      "queue.name": "q",
      "runtime-id": "dc862c6ff169400783e513c72f18a9f3",
      "span.kind": "producer"
    },
    "metrics": {
      "_dd.agent_psr": 1.0,
      "_dd.top_level": 1,
      "_dd.tracer_kr": 1.0,
      "_sampling_priority_v1": 1,
      "process_id": 55795
    },
    "duration": 2528372,
    "start": 1691167586717291958
  },
     {
       "name": "rq.worker.perform_job",
       "service": "rq-worker",
       "resource": "job_on_class",
       "trace_id": 1,
       "span_id": 2,
       "parent_id": 1,
       "type": "worker",
       "error": 0,
       "meta": {
         "_dd.p.dm": "-0",
         "component": "rq",
         "job.id": "9ccae7ee-b7c5-4189-a8a6-359ae2ed8570",
         "job.origin": "q",
         "job.status": "finished",
         "language": "python",
         "runtime-id": "dc862c6ff169400783e513c72f18a9f3",
         "span.kind": "consumer",
         "traceparent": "00-00000000000000005ccca5c9e5d71ed4-3c12ccb6011640bd-01",
         "tracestate": "dd=s:1;t.dm:-0"
       },
       "metrics": {
         "_dd.top_level": 1,
         "_dd.tracer_kr": 1.0,
         "_sampling_priority_v1": 1,
         "process_id": 55795
       },
       "duration": 1620811,
       "start": 1691167586726914941
     },
        {
          "name": "rq.job.perform",
          "service": "rq-worker",
          "resource": "job_on_class",
          "trace_id": 1,
          "span_id": 3,
          "parent_id": 2,
          "type": "",
          "error": 0,
          "meta": {
            "_dd.p.dm": "-0",
            "component": "rq",
            "job.id": "9ccae7ee-b7c5-4189-a8a6-359ae2ed8570"
          },
          "duration": 134555,
          "start": 1691167586727430938
        }]]
