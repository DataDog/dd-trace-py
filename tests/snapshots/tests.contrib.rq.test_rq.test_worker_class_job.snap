[[{"name" "rq.queue.enqueue_job"
   "service" "rq"
   "resource" "tests.contrib.rq.jobs.JobClass.job_on_class"
   "type" "worker"
   "error" 0
   "span_id" 0
   "trace_id" 0
   "parent_id" nil
   "start" 1630539839015211000
   "duration" 11457000
   "meta" {"runtime-id" "8140542a64974813b38ce3a795a83268"
           "queue.name" "q"
           "job.id" "3fed17e6-0034-416e-b7cc-ca41f69ad64e"
           "job.func_name" "job_on_class"}
   "metrics" {"_dd.agent_psr" 1.0
              "_sampling_priority_v1" 1
              "system.pid" 19332
              "_dd.tracer_kr" 1.0}}
      {"name" "rq.worker.perform_job"
       "service" "rq-worker"
       "resource" "job_on_class"
       "type" "worker"
       "error" 0
       "span_id" 1
       "trace_id" 0
       "parent_id" 0
       "start" 1630539839061506000
       "duration" 10969000
       "meta" {"runtime-id" "8140542a64974813b38ce3a795a83268"
               "job.id" "3fed17e6-0034-416e-b7cc-ca41f69ad64e"
               "job.status" "finished"
               "job.origin" "q"}
       "metrics" {"_sampling_priority_v1" 1
                  "system.pid" 19332
                  "_dd.tracer_kr" 1.0}}
          {"name" "rq.job.perform"
           "service" "rq-worker"
           "resource" "job_on_class"
           "error" 0
           "span_id" 2
           "trace_id" 0
           "parent_id" 1
           "start" 1630539839063398000
           "duration" 1287000
           "meta" {"job.id" "3fed17e6-0034-416e-b7cc-ca41f69ad64e"}}]
 [{"name" "rq.queue.enqueue_job"
   "service" "rq"
   "resource" "tests.contrib.rq.jobs.JobClass.__call__"
   "type" "worker"
   "error" 0
   "span_id" 0
   "trace_id" 1
   "parent_id" nil
   "start" 1630539839028374000
   "duration" 4931000
   "meta" {"runtime-id" "8140542a64974813b38ce3a795a83268"
           "queue.name" "q"
           "job.id" "6ae53cab-d198-4e1a-851a-71b0f853bc63"
           "job.func_name" "__call__"}
   "metrics" {"_dd.agent_psr" 1.0
              "_sampling_priority_v1" 1
              "system.pid" 19332
              "_dd.tracer_kr" 1.0}}
      {"name" "rq.worker.perform_job"
       "service" "rq-worker"
       "resource" "__call__"
       "type" "worker"
       "error" 0
       "span_id" 1
       "trace_id" 1
       "parent_id" 0
       "start" 1630539839102215000
       "duration" 9988000
       "meta" {"runtime-id" "8140542a64974813b38ce3a795a83268"
               "job.id" "6ae53cab-d198-4e1a-851a-71b0f853bc63"
               "job.status" "finished"
               "job.origin" "q"}
       "metrics" {"_sampling_priority_v1" 1
                  "system.pid" 19332
                  "_dd.tracer_kr" 1.0}}
          {"name" "rq.job.perform"
           "service" "rq-worker"
           "resource" "__call__"
           "error" 0
           "span_id" 2
           "trace_id" 1
           "parent_id" 1
           "start" 1630539839104129000
           "duration" 1286000
           "meta" {"job.id" "6ae53cab-d198-4e1a-851a-71b0f853bc63"}}]]
